CRM/Utils/Check/Component/Security.php

```diff
@@ -140,6 +144,28 @@ public function checkUploadsAreNotAccessible() {
     return $messages;
   }
 
+  /**
+   * Some security checks require sending a real HTTP request. This breaks the single-threading
+   * model historically used by the PHP built-in webserver (for local development). There is some
+   * experimental support for multi-threading in PHP 7.4+. Anecdotally, this is still insufficient
+   * on PHP 7.4 -- but it works well enough on PHP 8.1.
+   *
+   * @return CRM_Utils_Check_Message[]
+   */
+  public function checkHttpAuditable() {
+    $messages = [];
+    if ($this->isLimitedDevelopmentServer()) {
+      $messages[] = new CRM_Utils_Check_Message(
+        __FUNCTION__,
+        ts('In PHP 7.x, the built-in HTTP server cannot execute some security checks. This problem only affects local development on older versions of PHP.'),
+        ts('Incomplete Security Checks'),
+        \Psr\Log\LogLevel::ERROR,
```

Comment:
Was this intended as level ERROR? It causes a problem in front-end tests that have php 7.4 as part of the matrix since it interprets it the same as any other red popup and so the test fails.

Reply:
@demeritcowboy Sort of, but I'm ambivalent about the level - for the developer-user (*foreseen audience of 5e8a6bb2f034215be7b888871b61e4e422e525c3*), one can imagine rationales for ERROR or WARNING or NOTICE. I've downgraded it to WARNING in hopes that it avoids knock-on effects in other suites.

Tangentially - Are you pointing out a general/likely issue (*for any ERROR-level check*) or an observed regression (*with this specific check*)? My observation was that several `Security` checks hung on php 7.4.29 `cli-server` (macOS; Civi-standalone). But if there's some other php74 where those tests work, then that's an interesting data-point.